netdata/netdata
# Split chars (eBPF <-> Apps integration)

##### Summary
Update eBPF charts to have same pattern used with apps.plugin.

##### Test Plan
1. Change your configuration file `/etc/netdata/ebpf.d.conf` to:
```sh
[global]
    ebpf load mode = return
[ebpf programs]
    cachestat = yes
    dcstat = yes
    fd = yes
    process = yes
    shm = yes
    socket = yes
    swap = yes
    vfs = yes
```
2. Compile this branch
3. Start netdata confirming we have charts per app. You can also confirm this through our API accessing http://localhost:19999/api/v1/charts?all

##### Additional Information
<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? ebpf.plugin
- Can they see the change or is it an under the hood? If they can see it, where? Yes, now we won't have more one chart grouping apps data.
- How is the user impacted by the change? Explicit metrics.
- What are there any benefits of the change? We have now the same pattern used with apps and other software.
</details>
